packages/types/src/schemas/tutorial.ts

```diff
@@ -3,6 +3,7 @@ import { webcontainerSchema } from './common.js';
 
 export const tutorialSchema = webcontainerSchema.extend({
   type: z.literal('tutorial'),
+  logoLink: z.string().optional(),
```

Comment:
Not sure about the `logoLink` name. Should we just call it `homepage` or `home` instead? I'm fine with keeping it though. You know, naming...

Reply:
Yeah I found `logoLink` to be more explicit about what that property does but it doesn't look great.

I don't know what to think about `homepage` or `home` because you can have it point to anything like `https://buymeacoffee.com/...`.